libs/barista-components/button-group/src/button-group-item.scss

```diff
@@ -46,8 +46,3 @@
     color: $gray-300;
   }
 }
-
-// Apply the button-group-item theme to all theme definitions
-@include dt-apply-theme() {
-  @include dt-theme-button-group-item($dt-current-theme);
-}
```

Comment:
If you import the theme at the top, it will change the order of the selectors. 
before it was only including a mixin, that was called when the themes were compiled. To keep the source order, you should import the theme at the end of this file.